parm/wflow/verify_det.yaml

```diff
@@ -136,49 +136,60 @@ metatask_GridStat_REFC_RETOP_all_mems:
               age: 00:00:00:30
             text: !cycstr '{{ workflow.EXPTDIR }}/@Y@m@d@H/post_files_exist_mem#mem#.txt'
 
-metatask_PointStat_SFC_UPA_all_mems:
+metatask_PointStat:
   var:
-    mem: '{% if global.DO_ENSEMBLE %}{% for m in range(1, global.NUM_ENS_MEMBERS+1) %}{{ "%03d "%m }}{%- endfor -%} {% else %}{{ "000"|string }}{% endif %}'
-  metatask_PointStat_SFC_UPA_mem#mem#:
+    FIELD_GROUP: '{% for var in verification.VX_FIELD_GROUPS %}{% if var in ["SFC", "UPA", "AOD", "PM25", "PM10"] %}{{ "%s " % var }}{% endif %}{% endfor %}'
```

Comment:
It's cleaner and easier to read to use the one-liner Jinja if-statement in cases like these (and in the next lines):

```
{{ "%s " % var if var in ["SFC", "UPA", "AOD", "PM25", "PM10"] }}
```

There aren't as many `{% %}` blocks to keep up with.

Reply:
Fixed this one, but the rest are "if/else" statements so I don't think this solution works well